code/modules/lighting/lighting_process.dm

```diff
@@ -25,4 +25,4 @@
 
 		scheck()
 
-	lighting_update_overlays.Cut()
```

Comment:
Might want to tell the folks over at VG about the leak; this issue was just reported like...last week I believe; either case, len = 0 does outperform cut (BARELY), but the leak is pretty nasty.

Big thanks to RemieRichards for informing me of this.